prompts/2024/english/prompt_2024_01/AnaColon_summary_Bingham_and_Ratcliff_2024

```diff
@@ -0,0 +1,17 @@
+Complex multicellularity has evolved five separate times in eukaryotes but has never evolved in prokaryotes. Complex multicellularity has been defined as a large organism composed of many cells that lives for an extensive period of time. Some prokaryotes are multicellular but they do not have the morphologies of complex multicellular organisms. 
```

Comment:
To improve this writing for e.g. a blog post or professional summary, you'll want to make sure you don't start two back-to-back sentences the same way ('Complex multicellularity has...'). Or is the first sentence meant to be a title? If so, you should try out markdown formatting. First you'll have to change the filename as I suggested above (adding the `'.md'` extension tells GitHub to treat your text as if it's markdown instead of plain text). Then, you simply have to put a pound sign at the beginning of a line that's meant to act as a heading. E.g.:

Code:
`# Complex multicellularity has evolved...` 

Renders as
# Complex multicellularity has evolved...

You can see how it looks when rendered by using the 'preview' button when editing on GitHub